Formula/pspg.rb

```diff
@@ -0,0 +1,31 @@
+class Pspg < Formula
+  desc "Unix pager optimized for psql"
+  homepage "https://github.com/okbob/pspg"
+  url "https://github.com/okbob/pspg/archive/0.5.tar.gz"
+  sha256 "754d1e380d072517e9bc2c3c38785e2f19a9f927f061de9a646fd1094baa204e"
+
+  head do
```

Comment:
You don't need the block unless you're doing something special with `head` exclusively. Otherwise simply:
```ruby
head "https://github.com/okbob/pspg.git"
```
under the `sha256` line will suffice.

Reply:
Heh, I copy-pasted from another formula that had special build dependencies for `head`. Oops. Fixing.